packages/datadog-plugin-code-origin/src/fastify.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
```

Comment:
Not a blocker but more of a thought, I'm not sure about this structure of having a `code-origin` plugin. It makes sense from an ownership perspective, but makes it a bit difficult to navigate the Fastify plugin since it jumps to other places. With a centralized plugin manager this wouldn't be an issue, but from an existing plugin it can be surprising. This is especially true since right now every plugin folder is for a specific module, so from a reader it seems like there is a `code-origin` module that we support.

If you prefer to keep the code structure that is currently in this PR, I'd probably use a `datadog-code-origin` folder instead to hold the base plugin and also any other helper related to code origin.

Reply:
So are you saying you prefer this to be in the `datadog-plugin-fastify` folder again and not make it a composite plugin after-all, or have I misunderstood you?

Reply:
For now at least until this is improved with proper support for a different structure.

Reply:
Fixed, based on our Zoom call 👍